### DISCUSSION TITLE
Allow changing the raySize of LivingEntity#rayTraceEntities

### DISCUSSION THREAD

**Discussion by TheLimeGlass** [PaperMC/Paper#10596] [Ideas]
### Is your feature request related to a problem?

Allow changing the raySize of LivingEntity#rayTraceEntities
This will never include Displays because Displays have a hitbox of 0.

### Describe the solution you'd like.

Allowing the definition of the raySize so developers can include Displays.

### Describe alternatives you've considered.

Requiring Spigot's long method
```
result = origin.getWorld().rayTraceEntities(origin.getEyeLocation(), location.getDirection(), 100, 1.0D, entity -> !entity.equals(origin));
```

### Other

I told Spigot about this, and they're adding a JavaDoc to `World#rayTraceEntities` to tell developers to define the raySize for Displays. Rather than the method that inputs 0.0D for the raySize.

**Comment by emilyy-dev** [comment]
I think that we could introduce some sort of RayTraceBuilder, akin to the ParticleBuilder, to better streamline the process of ray tracing, rather than adding Yet Another Overload™️
